testsuite/integration/basic/pom.xml

```diff
@@ -1432,13 +1432,31 @@
             </plugin>
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
-                <artifactId>wildfly-jar-maven-plugin</artifactId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.org.wildfly.plugin}</version>
```

Comment:
The version shouldn't be needed here. If it is, we should define it somewhere higher up the parent chain in the `<pluginManagement>` section.

Reply:
@jamezp , I followed existing pattern (the version is defined everywhere), introduced minimal changes. I could refactor it in this PR. Perhaps in an extra commit.